tests/integration/helm_ot_logs.go

```diff
@@ -0,0 +1,43 @@
+//go:build allversions
```

Comment:
Instead of creating a special test execution, can we just put this feature in the default test? Or even make it a part of the existing multiline feature?

Reply:
We would have to craft fluent-bit configuration. Do we want that?

Reply:
I'd rather not. But we can still just add it to this test: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/tests/integration/helm_ot_default_test.go instead of creating a new one.